plugins/lighthouse/README.md

```diff
@@ -47,6 +47,10 @@ const routes = (
 Then configure the `lighthouse-audit-service` URL in your [`app-config.yaml`](https://github.com/backstage/backstage/blob/master/app-config.yaml).
 
 ```yaml
+backend:
```

Comment:
this actually wants a changeset - this readme gets shipped to npm - just something saying "Updated README" is fine